ride4dapps/momentaryLottery/momentaryLottery_test.js

```diff
@@ -0,0 +1,41 @@
+//
+// Momentary Lottery tests
```

Comment:
Add description that tests are run against current account. Current account should have at least xxx waves. Current account should hold some non waves token for test